highlight.io/components/common/Navbar/Navbar.tsx

```diff
@@ -43,6 +43,36 @@ const LaunchWeekBanner = () => {
 	return <Banner>{bannerMessage}</Banner>
 }
 
+const LivestreamBanner = () => {
+	return (
+		<Banner>
+			<div className={styles.launchWeekText}>
+				Distributed Tracing livestream: April 11 at 2pm PDT. Register
+				for&nbsp;
+				<a
+					target="_blank"
+					href="https://lu.ma/b0uz0fiz"
+					className={styles.launchWeekText}
+					style={{ textDecoration: 'underline' }}
+					rel="noreferrer"
+				>
+					Zoom
+				</a>
```

Comment:
could you please use the `<Link>` tag with the same styling as the startup banner below?
see 
```
<Link
						href="/xxx"
						className="flex justify-center items-center w-full h-[40px] bg-color-primary-200 text-white hover:bg-opacity-90"
					>
```

Reply:
I think I got it!